miner_config/generate_config.py

```diff
@@ -54,14 +54,24 @@ def output_config_file(config, path):
     open(path, "w").write(config)
 
 
+def is_testnet_fleet():
+    return bool(int(os.getenv('TESTNET', '0')))
+
+
+def is_rockpi():
+    return bool(int(os.getenv('ROCKPI', '0')))
+
+
 def main():
     init_sentry()
-    if bool(int(os.getenv('PRODUCTION', '0'))):
-        base_url = 'https://helium-snapshots.nebracdn.com'
-    else:
+    if is_testnet_fleet():
```

Comment:
For consistency with [other code](https://github.com/NebraLtd/hm-block-tracker/blob/master/snapshotter/create_snapshot.sh#L7), to reduce the number of env vars, and to make testnet/staging default, I suggest:

```
if is_production():
  # prod logic
else:
  # staging logic
```

Reply:
done